subsys/zigbee/cli/zigbee_cli_cmd_zdo.c

```diff
@@ -272,9 +272,17 @@ static void cmd_zb_simple_desc_req_cb(zb_bufid_t bufid)
 			simple_desc_resp->simple_desc.app_device_id,
 			simple_desc_resp->simple_desc.app_device_version);
 
+		/* Print what allready present in buffer and reset it. */
+		shell_fprintf(ctx_entry->shell, SHELL_NORMAL, "%s", text_buffer);
+		text_buffer[0] = '\0';
```

Comment:
Why is this better than increasing the `text_buffer` size?

Reply:
No particular reason, increasing `text_buffer` should work as well

Reply:
With `text_buffer` increased to 256, should be sufficient for up to 20 cluster (in summary)